index.ts

```diff
@@ -214,18 +210,25 @@ async function exportSingleEvent(
         customerPayload.created_at = Date.parse(customerPayload.created_at) / 1000
     }
 
-    let id = event.distinct_id
+    let id = event.person.uuid
 
     if (customer.email) {
         customerPayload.email = customer.email
         if (identifyByEmail) {
             id = customer.email
         }
     }
+
     // Create or update customer
     // See https://www.customer.io/docs/api/#operation/identify
     await callCustomerIoApi('PUT', host, `/api/v1/customers/${id}`, authorizationHeader, customerPayload)
 
+    if (['$identify', '$merge_dangerously'].includes(event.event) && !event.$merge_blocked) {
+        const secondaryId = event?.properties?.$anon_distinct_id ?? event?.properties?.alias
+        const mergeUserPayload = { primary : { id: event.distinct_id }, secondary: { id: secondaryId }}
```

Comment:
how does this work with the person.uuid usage change above?

Reply:
Yeah this only works on the first identify, since the person UUID inside of PostHog isn't changing, but for additional identifies or merge_dangerously. We'll need to convert this `distinct_id` into the PostHog UUID. Should I just call `http://localhost:8000/api/projects/1/persons?distinct_id=9fe3478a-3e19-4c7b-8c93-f0bc32b45341` inside of PostHog? That will require me to have some kind of access as well.